### PR TITLE
Fix GoToEvent link generation when value is None

### PR DESCRIPTION
**Pull request by hramezani**

### CODE REVIEW

File: src/npm-fastui/src/components/display.tsx [hunk 1]
```diff
@@ -218,9 +218,7 @@ const subKeys = (template: string, row: DataModel): string | null => {
   let returnNull = false
   const r = template.replace(/{(.+?)}/g, (_, key: string): string => {
     const v: JsonData | undefined = row[key]
-    if (v === undefined) {
-      throw new Error(`field "${key}" not found in ${JSON.stringify(row)}`)
-    } else if (v === null) {
+    if (v === null || v === undefined) {
       returnNull = true
       return 'null'
```
Comment (sydney-runkle) [inline src/npm-fastui/src/components/display.tsx:223]: Why do we return `'null'` here and `null` below? Do we no longer need to raise an error?

Comment (hramezani) [reply src/npm-fastui/src/components/display.tsx:223]: > Why do we return 'null' here and null below?

Don't know.

> Do we no longer need to raise an error?

I think we should keep it like that. I will change the implementation

**Comment by hramezani** [comment]
Ok, I changed the implementation.

The table renders without any error with my patch. you can see the row begins with `Tokyo` is not rendered as a link (because I changed the data to reproduce the problem and confirm my fix)

image

but it fails without my patch:

image

**Review by sydney-runkle — APPROVED** [review]
Looks great, thanks @hramezani

**Pull request merged**